composer.json

```diff
@@ -53,59 +68,31 @@
   },
   "require-dev": {
     "doctrine/doctrine-fixtures-bundle": "^3.4",
-    "malukenho/docheader": "^0.1.6",
-    "mockery/mockery": "^1.5.1",
     "php-parallel-lint/php-parallel-lint": "^1.3",
+    "malukenho/docheader": "^0.1.6",
+    "mockery/mockery": "^1.2",
     "phpmd/phpmd": "^2.13",
     "phpunit/phpcov": "~8.2.1",
     "phpunit/phpunit": "~9.5.26",
     "sebastian/phpcpd": "^6.0.3",
-    "squizlabs/php_codesniffer": "^2.8",
+    "squizlabs/php_codesniffer": "^3.7",
     "symfony/css-selector": "5.4.*",
     "symfony/phpunit-bridge": "5.4.*",
     "symfony/stopwatch": "5.4.*",
     "symfony/web-profiler-bundle": "5.4.*"
   },
-  "config": {
-    "allow-plugins": {
-      "composer/package-versions-deprecated": true,
-      "symfony/flex": true
-    },
-    "preferred-install": {
-      "*": "dist"
-    },
-    "sort-packages": true
-  },
-  "autoload": {
-    "psr-4": {
-      "Surfnet\\ServiceProviderDashboard\\": "src/Surfnet/ServiceProviderDashboard"
-    }
-  },
-  "autoload-dev": {
-    "psr-4": {
-      "Surfnet\\ServiceProviderDashboard\\": "src/Surfnet/ServiceProviderDashboard",
-      "Surfnet\\ServiceProviderDashboard\\Webtests\\": "tests/webtests",
-      "Surfnet\\ServiceProviderDashboard\\Tests\\": "tests/"
-    }
-  },
-  "replace": {
-    "paragonie/random_compat": "2.*",
-    "symfony/polyfill-ctype": "*",
-    "symfony/polyfill-iconv": "*",
-    "symfony/polyfill-php71": "*",
-    "symfony/polyfill-php70": "*",
-    "symfony/polyfill-php56": "*"
-  },
   "scripts": {
-    "auto-scripts": {
-      "cache:clear": "symfony-cmd",
-      "assets:install %PUBLIC_DIR%": "symfony-cmd"
-    },
+    "symfony-scripts": [
+      "Incenteev\\ParameterHandler\\ScriptHandler::buildParameters",
```

Comment:
This Symfony script will not work anymore as we removed the parameters handler dependency.